### PR TITLE
Fix XmfHtml Gump element to phase args issues #37

### PR DESCRIPTION
**Pull request by Nirad**
Fix for issues #37 

Before :
capture d ecran 2019-02-15 a 14 46 16

After:

capture d ecran 2019-02-15 a 14 33 33
capture

### CODE REVIEW

File: src/Managers/PacketManager.cpp [hunk 1]
```diff
@@ -4878,6 +4879,7 @@ PACKET_HANDLER(OpenGump)
     for (const string &str : commandList)
     {
         vector<string> list = cmdParser.GetTokens(str.c_str());
+        vector<string> listNt = cmdParser.GetTokens(str.c_str(), false);
```
Comment (fungos) [inline src/Managers/PacketManager.cpp]: It is not good to to two times the same work. What the trim caused on the first list that you couldn't use it or make it not trim?

File: src/Managers/ClilocManager.cpp [hunk 2]
```diff
@@ -277,3 +276,59 @@ wstring CClilocManager::ParseArgumentsToClilocString(int cliloc, bool toCamelCas
 
     return message;
 }
+
+wstring CClilocManager::ParseXmfHtmlArgumentsToCliloc(int cliloc, bool toCamelCase, wstring args)
```
Comment (fungos) [inline src/Managers/ClilocManager.cpp]: This is basically a copy of `ParseArgumentsToClilocString` and only the start of the function changes (the while erase part). Then to make it a bit better, do a 3rd private function with only the code after the first while, and do these two functions call the new one passing the new cleared args, unless there is any other differences between they that I'm not seeing right now.

File: src/Managers/PacketManager.cpp (hunk 1 above)
Comment (Nirad) [reply src/Managers/PacketManager.cpp]: the trim one remove /t from arg so I can not phase the after

Comment (Nirad) [reply src/Managers/PacketManager.cpp]: i will test if I can use only one not trim

**Comment by Nirad** [comment]
all sold be good! thx

**Pull request merged**